runtime/ops/os/mod.rs

```diff
@@ -386,6 +386,39 @@ fn rss() -> usize {
   task_info.resident_size as usize
 }
 
+#[cfg(target_os = "openbsd")]
+fn rss() -> usize {
+  let pid = unsafe { libc::getpid() };
+  let pagesize = unsafe { libc::sysconf(libc::_SC_PAGESIZE) } as usize;
+  let mut kinfoproc: std::mem::MaybeUninit<libc::kinfo_proc> =
+    std::mem::MaybeUninit::uninit();
```

Comment:
**nitpick:** I think you can avoid the double-`std::mem::MaybeUninit` with
```rs
let mut kinfoproc = std::mem::MaybeUninit<libc::kinfo_proc>::uninit();
```

Reply:
Nice! Done.